src/config/security/hashboot.md

```diff
@@ -0,0 +1,55 @@
+# Hashboot
+
+**hashboot** hashes all files in `/boot`, the MBR to check them during early
+boot. It is intended for when the root partition is encrypted but not the boot
+partition. The checksums and a backup of the contents of `/boot` are stored in
+`/var/lib/hashboot` by default. If a checksum doesn't match, there is the option
+to restore the file from backup.
+
+If there is a core- or libreboot bios, **hashboot** can check bios for
+modifications too.
+
+## Installation
+
+```
+# xbps-install -S hashboot
+```
+
+To verify BIOS, `flashrom` needs to be installed, too.
+
+## Configuration
+
+After installation run
+
+```
+# hashboot index
+```
+
+to compose the configurationfile and generate the index of the chosen options.
+
+> If this is not run after installation, next boot will stop with an emergency
+> shell.
+
+## Options
+
+The options are chosen by a bitmask:
```

Comment:
How are these options passed to hashboot?

Reply:
Should be clear now.